src/Common/Throttler.h

```diff
@@ -61,12 +61,14 @@ class Throttler
     UInt64 getMaxSpeed() const { return static_cast<UInt64>(max_speed); }
     UInt64 getMaxBurst() const { return static_cast<UInt64>(max_burst); }
 
+    void setMaxSpeed(size_t max_speed_);
+
 private:
     void addImpl(size_t amount, size_t & count_value, double & tokens_value);
 
     size_t count{0};
-    const size_t max_speed{0}; /// in tokens per second.
-    const size_t max_burst{0}; /// in tokens.
+    mutable size_t max_speed{0}; /// in tokens per second.
+    mutable size_t max_burst{0}; /// in tokens.
```

Comment:
why mutable?

Reply:
just want to update the value, i will use non const to instead of mutable